recipes_source/recipes_index.rst

```diff
@@ -324,6 +324,13 @@ Recipes are bite-sized, actionable examples of how to use specific PyTorch featu
    :link: ../recipes/DCP_tutorial.html
    :tags: Distributed-Training
 
+   .. customcarditem::
+   :header: Getting Started with DeviceMesh
+   :card_description: Learn how to use DeviceMesh
+   :image: ../_static/img/thumbnails/cropped/profiler.png
+   :link: ../recipes/distributed_device_mesh.html
+   :tags: Distributed-Training
```

Comment:
shall we put this card to the first of the distributed section? iirc all other receipts are not quite useful anymore (i.e. RPC)

Reply:
> shall we put this card to the first of the distributed section? iirc all other receipts are not quite useful anymore (i.e. RPC)

We can add the tutorial on the distributed page as @svekars  suggested: https://pytorch.org/tutorials/distributed/home.html